ibis/backends/pandas/execution/window.py

```diff
@@ -361,15 +371,16 @@ def execute_window_op(
         clients=clients,
         **kwargs,
     )
-    series = post_process(
+    computed = post_process(
```

Comment:
calling this `window_result` is a little better I think

Also fine to reuse `result` as well

Reply:
Sure, I will keep using `result`